### PR TITLE
Interop KYC and Interop Loan Disbursement

### PR DESCRIPTION
**Pull request by SubhamPramanik**
## Description

Describe the changes made and why they were made.
- InteropAPI to get client KYC using account ID
- Interop API to disburse loan

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [x] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by avikganguly01** [comment]
@SubhamPramanik Can you fix the checkstyle violations?

File: fineract-provider/src/main/java/org/apache/fineract/interoperation/api/InteropApiResource.java [hunk 1]
```diff
@@ -373,4 +374,29 @@ public String performTransfer(@QueryParam("action") @Parameter(description = "ac
...
+    @GET
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Path("accounts/{accountId}/kyc")
+    @Operation(summary = "Query KYC by Account Id", description = "")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = InteropKycResponseData.class))) })
+    public String getClientKyc(@PathParam("accountId") @Parameter(description = "accountId") String accountId, @Context UriInfo uriInfo) {
+        InteropKycResponseData result = interopService.getKyc(accountId);
+        ApiRequestJsonSerializationSettings settings = this.apiRequestParameterHelper.process(uriInfo.getQueryParameters());
```
Comment (avikganguly01) [inline fineract-provider/src/main/java/org/apache/fineract/interoperation/api/InteropApiResource.java:387]: Check permission for this action and entity (READ - CLIENTIDENTIFIER) ...context.authenticatedUser().validateHasReadPermission(ENTITY_NAME);

File: fineract-provider/src/main/java/org/apache/fineract/interoperation/service/InteropServiceImpl.java [hunk 2]
```diff
@@ -437,6 +550,19 @@ private SavingsAccount validateAndGetSavingAccount(@NotNull InteropRequestData r
         return savingsAccount;
     }
 
+    // private Loan validateAndGetLoan(@NotNull InteropRequestData request) {
```
Comment (avikganguly01) [inline fineract-provider/src/main/java/org/apache/fineract/interoperation/service/InteropServiceImpl.java]: Remove this if you're not planning to use it.

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

**Comment by thesmallstar** [comment]
@SubhamPramanik  thank you for the PR, I see a checkstyle error in your code, that means address_line_1 that you made does not match the required regex it should.

I also see a AvoidHidingCauseException, that happens when you catch a exception and throw another exception essentially hiding what actually caused the exception! 

Will you fix this?

cc @vorburger The checkstyles are helping!!

File: fineract-provider/src/main/java/org/apache/fineract/interoperation/service/InteropServiceImpl.java (hunk 2 above)
Comment (SubhamPramanik) [reply fineract-provider/src/main/java/org/apache/fineract/interoperation/service/InteropServiceImpl.java]: Removed

File: fineract-provider/src/main/java/org/apache/fineract/interoperation/api/InteropApiResource.java (hunk 1 above)
Comment (SubhamPramanik) [reply fineract-provider/src/main/java/org/apache/fineract/interoperation/api/InteropApiResource.java:387]: Not sure how to do it. Can you point to a refernce implementation?

**Comment by vorburger** [comment]
@avikganguly01 looks like the review feedback from you here has been addressed? Will you re-review and merge when OK for you? Please use _Squash and Merge_ (notice the drop-down on the green button above), as per https://github.com/apache/fineract#merge-strategy.

**Pull request merged**